src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorIntegrationTest.java

```diff
@@ -422,7 +451,9 @@ public void testCanAnnotateMixedContigHg19Clinvar() {
         arguments.addArgument(FuncotatorArgumentDefinitions.DATA_SOURCES_PATH_LONG_NAME, DS_PIK3CA_DIR);
         arguments.addArgument(FuncotatorArgumentDefinitions.REFERENCE_VERSION_LONG_NAME, FuncotatorTestConstants.REFERENCE_VERSION_HG19);
         arguments.addArgument(FuncotatorArgumentDefinitions.OUTPUT_FORMAT_LONG_NAME, outputFormatType.toString());
-        arguments.addBooleanArgument(FuncotatorArgumentDefinitions.ALLOW_HG19_GENCODE_B37_CONTIG_MATCHING_LONG_NAME, true);
+
+        // We need this argument since we are testing on a subset of b37
+        arguments.addBooleanArgument(FuncotatorArgumentDefinitions.FORCE_B37_TO_HG19_REFERENCE_CONTIG_CONVERSION, true);
```

Comment:
If your B37/HG19 detection mechanism were a bit more permissive, you might not need these shenanigans. In particular, if you require only 1 autosome to match a known contig+length from B37 or HG19, instead of requiring all contigs to match, then I think that you should be able to work with these subsetted test files without hacks like this.

Reply:
Being permissive with references already makes me uncomfortable.  I'm of the opinion that files should match in their entirety and that subsets aren't sufficient.  I'd prefer not to put in any more fuzzy matching with references.

As an aside, the reference issue in Funcotator should be solved by adding in another reference set for `b37` so that every reference has its own backing data sources.  This patch to me is something of a stop-gap to make Funcotator fast in the mean time.

Reply:
Good to know this workaround will be short-lived.